docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,430 @@
+from superlinked.framework.common.schema.id_schema_object import IdField
+from superlinked.framework.common.parser.dataframe_parser import DataFrameParser
+from superlinked.framework.dsl.executor.in_memory.in_memory_executor import (
+    InMemoryExecutor,
+    InMemoryApp,
+)
+from superlinked.framework.dsl.index.index import Index
+from superlinked.framework.dsl.source.in_memory_source import InMemorySource
+from superlinked.framework.dsl.space.text_similarity_space import TextSimilaritySpace
+from superlinked.framework.dsl.space.number_space import NumberSpace
+from superlinked.framework.dsl.space.recency_space import RecencySpace
+
+
+alt.renderers.enable("mimetype")  # comment this line out when running in Colab to render altair plots
+alt.data_transformers.disable_max_rows()
+os.environ["TOKENIZERS_PARALLELISM"] = "false"
+pd.set_option("display.max_colwidth", 190)
+pd.options.display.float_format = "{:.2f}".format
+```
+
+Now we import our dataset. 
+
+```python
+dataset_repository_url = (
+    "https://storage.googleapis.com/superlinked-notebook-user-acquisiton-analytics"
+)
+USER_DATASET_URL = f"{dataset_repository_url}/user_acquisiton_data.csv"
+NOW_TS = 1708529056
+EXECUTOR_DATA = {CONTEXT_COMMON: {CONTEXT_COMMON_NOW: NOW_TS}}
+```
+
+(If you're waiting something you're executing to finish, you can always find interesting reading in [VectorHub](https://superlinked.com/vectorhub/).)
+
+## Read and explore our dataset
+
+Now that our dataset's imported, let's take a closer look at it:
+
+```python
+NROWS = int(os.getenv("NOTEBOOK_TEST_ROW_LIMIT", str(sys.maxsize)))
+user_df: pd.DataFrame = pd.read_csv(USER_DATASET_URL, nrows=NROWS)
+print(f"User data dimensions: {user_df.shape}")
+user_df.head()
+```
+
+We have 8000 users and (as we can see from the first five rows) 4 columns of data:
+
+![first 5 rows of our dataset](/user_id-sign_up_date-ad_creative-activity.png)
+
+To understand which ad creatives generated how many signups, we create a DataFrame:
+
+```python
+pd.DataFrame(user_df["ad_creative"].value_counts())
+```
+
+which looks like this:
+
+![ad_creatives by count](/ad_creative-count.png)
+
+observation: the influencer (XYZCr$$d) backed ad creatives seem to have worked better - generating many more [signups?] than the August ad creatives.
+
+Now, let's take a look at the distribution of users according to activity level.
+
+```python
+alt.Chart(user_df).mark_bar().encode(
+    alt.X("activity:Q", bin=True, title="Activity count"),
+    alt.Y("count()", title="# of users"),
+).properties(width=600, height=400)
+```
+
+![distribution by activity count](/user_distrib-activity_count.png)
+
+out[6]: activity = api calls / day?
+The activity distribution is bimodal. Here, the first activity level group may be largely new users, who - because they signed up in the most recent (December) campaign - have less time to accumulate activity than users who signed up earlier.
+Also, this informs NumberSpace parameters[=? how?]
+
```

Comment:
@morkapronczay
"this informs NumberSpace parameters" - how?

Reply:
NumberSpace min and max can be derived from the distribution